documentation/docs/workflow.md

```diff
@@ -7,113 +7,141 @@ keywords:
 - Workflow
 - Contribute
 - GitHub
+- explanation
 ---
+# Identity Workflow
+
+In this article you will learn about the software development process for the IOTA Identity repository as well as key terms, functions, and the overall operability of the workflow components.
+
 ## Issues
 
-Issues are opened when a certain task or problem is noted but cannot immediately be fixed. Issues may contain bug reports, requests, or larger topics. Please use the correct GitHub issue template for your issue type. Only IOTA Foundation members should use the task templates flagged for maintainers. Make sure to [label](#Issue-Labels) the issue correctly. As a contributor you may also add issues to a certain [project](https://github.com/iotaledger/identity.rs/projects/).
+Issues are opened when a certain task or problem is noted but cannot immediately be fixed. Issues may contain bug reports, requests, or larger topics. Please use the correct GitHub issue template for your issue type. Only IOTA Foundation members should use the task templates flagged for maintainers. You should make sure to [label](#Issue-Labels) the issue correctly. As a contributor, you may also add issues to a certain [project](https://github.com/iotaledger/identity.rs/projects/).
 
 ## Git
 
 ### Pull Requests
 
-New branches should be pushed to the GitHub repository as soon as possible, making them public to all contributors. In addition, a pull request (PR) should be opened in draft status, describing the goals and any requirements of the changes. In order to generate good [changelogs](#Changelog), the PR title must be written in a way that is suitable as an changelog entry and the PR needs to be [labeled](#PR-Labels) correctly.
+New branches should be pushed to the GitHub repository as soon as possible, making them public to all contributors. In addition, a pull request (PR) should be opened in draft status, describing the goals and any requirements of the changes. To generate good [changelogs](#Changelog), a PR title must be written in a way that is suitable as a changelog entry while the PR must be [labeled](#PR-Labels) correctly.
```

Comment:
I think we can put the user at the center in the entire workflow section.  "You should push new branches ...., . You should open a pull request in draft status....etcetc